src/components/content-switcher/content-switcher.scss

```diff
@@ -13,6 +13,14 @@ $css--plex: true !default;
   @extend .#{$prefix}--content-switcher;
 }
 
+:host(#{$prefix}-content-switcher[size='sm']) {
+  height: rem(32px);
+}
+
+:host(#{$prefix}-content-switcher[size='xl']) {
+  height: rem(48px);
```

Comment:
```suggestion
  @extend .#{$prefix}--content-switcher--xl;
```

Reply:
@asudoh Good call! Interesting that this works, in the past I've had problems extending classes directly from other imports.

Reply:
@kennylam Yeah it has to be a simple selector I believe but where it works we can avoid CSS rule duplicates.